ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -264,7 +272,37 @@ QObject {
 
         sdk: root.wcSDK
         store: root.store
-        supportedAccountsModel: d.supportedAccountsModel
+        supportedAccountsModel: SortFilterProxyModel {
```

Comment:
check in 'disconnect' if proper dApp from proper wallet address is disconnected.

Reply:
Disconnect works correctly